### PR TITLE
Test with Travis

### PR DESCRIPTION
**Pull request by adamchainz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+notifications:
+  email: false
+
+dist: trusty
+sudo: false
+language: python
+python: 2.7
```
Comment (MichaelAquilina) [inline .travis.yml]: you can use `tox-travis` and parrallelise your builds https://github.com/MichaelAquilina/S4/blob/master/.travis.yml

Comment (adamchainz) [inline .travis.yml]: I don't

Comment (adamchainz) [inline .travis.yml]: In many cases for small projects I found the serial version is faster

Comment (MichaelAquilina) [inline .travis.yml]: Makes sense 👍

**Pull request merged**